src/renderer/processes/multisigs/model/multisigs-model.ts

```diff
@@ -51,7 +51,7 @@ type GetMultisigsResult = {
 const getMultisigsFx = createEffect(({ chains, wallets }: GetMultisigsParams) => {
   for (const chain of chains) {
     const filteredWallets = walletUtils.getWalletsFilteredAccounts(wallets, {
-      walletFn: (w) => !walletUtils.isMultisig(w) && !walletUtils.isWatchOnly(w) && !walletUtils.isProxied(w),
+      walletFn: (w) => !walletUtils.isMultisig(w) && !walletUtils.isProxied(w),
     });
```

Comment:
If we don’t exclude watchOnly, is it possible that we could receive multisig wallets for those as well?

Reply:
That was for debug purposes(((